ops/Dockerfile.opnode

```diff
@@ -7,10 +7,16 @@ COPY go.mod /app/go.mod
 COPY go.sum /app/go.sum
 
 WORKDIR /app
+COPY ./ops/opnode-entrypoint.sh /app/opnode-entrypoint.sh
+COPY ./ops/rollup.json /app/rollup.json
+
 COPY ./opnode /app/opnode
 RUN make opnode
 
 FROM alpine:3.13
```

Comment:
This still needs to be updated to `alpine:3.15` unless it doesn't work

Reply:
Then caching will work better with the other monorepo

Reply:
Done